tests/pester-tests/powershell-commands/Uninstall-ChocolateyPath.Tests.ps1

```diff
@@ -1,12 +1,38 @@
-﻿Describe 'Uninstall-ChocolateyPath helper function tests' -Tags Cmdlets {
+﻿Describe 'Uninstall-ChocolateyPath helper function tests' -Tags UninstallChocolateyPath, Cmdlets {
     BeforeAll {
         Initialize-ChocolateyTestInstall
 
         $testLocation = Get-ChocolateyTestLocation
         Import-Module "$testLocation\helpers\chocolateyInstaller.psm1"
     }
 
-    Context 'Adding and removing PATH values' -ForEach @(
+    Context 'Unit tests' -Tags WhatIf -ForEach @(
+        @{ Scope = 'Process' }
+        @{ Scope = 'User' }
+        @{ Scope = 'Machine' }
+    ) {
+        It 'removes a stored PATH value in the desired PATH scope' {
+            $targetPathEntry = [Environment]::GetEnvironmentVariable('PATH', $Scope) -split ';' | Select-Object -First 1
+            $Preamble = [scriptblock]::Create("Import-Module '$testLocation\helpers\chocolateyInstaller.psm1'")
+            $Command = [scriptblock]::Create("Uninstall-ChocolateyPath -Path '$targetPathEntry' -Scope $Scope -WhatIf")
+            
+            $results = @( Get-WhatIfResult -Preamble $Preamble -Command $Command )
+            $results[0] | Should -BeExactly "What if: Performing the operation ""set $Scope environment variable"" on target ""PATH""."
+
+            if ($Scope -ne 'Process') {
+                $results[1] | Should -BeExactly 'What if: Performing the operation "Notify system of changes" on target "Environment variables".'
+                $results[2] | Should -BeExactly 'What if: Performing the operation "refresh all environment variables" on target "current process".'
+            }
+        }
+
+        It 'skips removing the value if it is not present' {
+            $targetPathEntry = [Environment]::GetEnvironmentVariable('PATH', $Scope) -split ';' | Select-Object -First 1
+            $Command = [scriptblock]::Create("Uninstall-ChocolateyPath -Path 'C:\ThisShouldNotBePresent' -Scope $Scope -WhatIf")
+            Get-WhatIfResult -Preamble $Preamble -Command $Command | Should -BeNullOrEmpty -Because 'we should skip adding values that already exist'

```

Comment:
Should the `Because` here not be something like `we should skip uninstalling a value that doesn't exist`?

Reply:
Good catch, thanks!